src/appl/rmcdhf90_mem/BUILDCONF.sh

```diff
@@ -0,0 +1,74 @@
+EXE=rmcdhf_mem
+LIBRARIES="dvd90 9290 mod"
+LAPACK=true
+FILES="
+mpi_s.f90
+
+dsubrs.f90 dsubrs_I.f90
+fco.f90 fco_I.f90
+gco.f90 gco_I.f90
+setcof.f90 setcof_I.f90
+lodcsh2GG.f90 lodcsh2GG_I.f90
+setcsl.f90 setcsl_I.f90
+setdbg.f90 setdbg_I.f90
+setham.f90 setham_I.f90
+xpot.f90 xpot_I.f90
+ypot.f90 ypot_I.f90
+dacon.f90 dacon_I.f90
+dampck.f90 dampck_I.f90
+setlag.f90 setlag_I.f90
+setmcp.f90 setmcp_I.f90
+setmix.f90 setmix_I.f90
+setsum.f90 setsum_I.f90
+setxuv.f90 setxuv_I.f90
+setxv.f90 setxv_I.f90
+setxz.f90 setxz_I.f90
+lagcon.f90 lagcon_I.f90
+cofpot.f90 cofpot_I.f90
+consis.f90 consis_I.f90
+csfwgt.f90 csfwgt_I.f90
+dampor.f90 dampor_I.f90
+defcor.f90 defcor_I.f90
+eigen.f90 eigen_I.f90
+engoutgg.f90 engoutgg_I.f90
+endsum.f90 endsum_I.f90
+estim.f90 estim_I.f90
+getaldwt.f90 getaldwt_I.f90
+getald.f90 getald_I.f90
+prtrsl.f90 prtrsl_I.f90
+getoldwt.f90 getoldwt_I.f90
+getold.f90 getold_I.f90
+getscd.f90 getscd_I.f90
+hmout.f90 hmout_I.f90
+out.f90 out_I.f90
+in.f90 in_I.f90
+prwf.f90 prwf_I.f90
+ispar.f90 ispar_I.f90
+itjpo.f90 itjpo_I.f90
+maneig.f90 maneig_I.f90
+matrix.f90 matrix_I.f90
+maxarr.f90 maxarr_I.f90
+newco.f90 newco_I.f90
+outbnd.f90 outbnd_I.f90
+newe.f90 newe_I.f90
+orbout.f90 orbout_I.f90
+orthor.f90 orthor_I.f90
+orthy.f90 orthy_I.f90
+solve.f90 solve_I.f90
+improv.f90 improv_I.f90
+scf.f90 scf_I.f90
+strsum.f90 strsum_I.f90
+read02_mem.f90 read02_mem_I.f90
+read03_mem.f90 read03_mem_I.f90
+read1_mem.f90 read1_mem_I.f90
+read2_mem.f90 read2_mem_I.f90
+read3_mem.f90 read3_mem_I.f90
+rmcdhf_mem_C.f90
```

Comment:
The error seems to be that the compiled `rmcdhf_mem_C` module is not found: https://github.com/compas/grasp/runs/4871824355?check_suite_focus=true#step:4:2081

I reckon it's because Make tries to compile stuff that depend on this module before actually compiling it. I suggest moving this line all the way to the top, so that it would get compiled first by Make.

As for why it works locally: did you try deleting all the build files? I guess you have the `.mod` file lying around from a previous compilation.

Reply:
Yup, you are right. Not sure how but somehow the .mod files were never cleaned out properly in my tests. Seems to be working now after some shuffling around. And good I learned the procedure of generating/updating the Makefiles with your genbuild scripts!